server/src/schema.rs

```diff
@@ -277,6 +381,30 @@ table! {
     }
 }
 
+table! {
+    private_message_fast (fast_id) {
```

Comment:
This table doesnt need to exist imo, it's not aggregating anything so its not faster than a normal query on the underlying records.  If we want the api to interface a common table names then its better to have views sitting on top the fast tables to abstract away dealing w/ table names